.github/workflows/golang_docker.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     - name: Build binary
       run: |
-        go build  -tags=viper_bind_struct -o keployv2
+        go build -race -tags=viper_bind_struct -o keployv2
 
```

Comment:
incase of docker we run via docker image of keploy, binary is just needed to start the said docker image thus I didn't put the flag in pipeline as it wasn't needed.